### ISSUE TITLE
[BUG] No Genre?

### ISSUE THREAD

**Issue by etnemo** [Depressurizer/Depressurizer#252]
Hello there,

I am using the latest relase. When I try to scrap my games, Tags are get scrapped but no Genres. Even, no genres on "Edit AutoCats" window. Is this a bug or am I doing something wrong?

**Comment by lanedenson** [comment]
I'm encountering the same issue

**Comment by TheBrokenG0d** [comment]
Just wanted to confirm its broken for several older versions.
5.3.0, 5.2.0, 4.13.1

Everything works except Genre.

**Comment by AbteuU** [comment]
Same here with latest version, now all my games are uncategorized since i allowed resetting my collections in Steam. I don't want to use tags but they have to do for now.

Edit: Not even Tags work. I'm either too dumb or it's simply not working.

**Comment by rbfussell** [comment]
Probably duplicate of https://github.com/Depressurizer/Depressurizer/issues/229   So 2 1/4 year of no fix.   Given that Genre is the most COMMON categorization method... seems somewhat necessary.  This really should be addressed.   There's a pull that seems to have been sitting in flux since september of last year that would fix it.  https://github.com/Depressurizer/Depressurizer/pull/256  but alas seems development has stalled to the point that not even merging simple fixes for major bugs that other's made.

**Comment by NigeNigeNige** [comment]
Depressurizer just nuked all of my categories because of this bug.

**Comment by livejamie** [comment]
Is there a workaround for this? A previous version or a fork that works?

**Comment by TheBrokenG0d** [comment]
> Is there a workaround for this? A previous version or a fork that works?

I've started to use Steam Dynamic Collections, they work great and have a Genre selection so I just created a collection for each Genre and I'm done and any new games are auto added to them.

**Comment by Tythesly** [comment]
Yeah everybody knows about that but you can't sort by year or (when it was
working) howlongtobeat times

**Comment by zenorach** [comment]
> Same here with latest version, now all my games are uncategorized since i allowed resetting my collections in Steam. I don't want to use tags but they have to do for now.
> 
> Edit: Not even Tags work. I'm either too dumb or it's simply not working.

I found that tags worked AFTER going in Database Editor (under tools) and then scraping all items.
But both genre and "flags" data is empty (flags is stuff like single-player, controller support)
